packages/ui-react/src/utils/componentProps.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/ban-types */
+
+export type CommonComponentProps<
```

Comment:
Can you add documentation, what is this type and when it should be used?

Reply:
Most likely, I will remove this at all. It is needed just for migration purposes.